docs/cristin.ttl

```diff
@@ -260,39 +286,39 @@
     :isignored   "true" .
 
 :arstall_rapportert
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment "Year that the resource was included in an NVI report." ;
     rdfs:range   xsd:positiveInteger ;
     rdfs:domain  :Result ;
     :isignored   "true" .
 
 :merknadtekst
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment "Free text for super-users to add comments when it comes to NVI approvals." ;
     rdfs:range   xsd:string ;
     rdfs:domain  :Result ;
     :isignored   "true" .
 
 :merknadtekst_godkjenning
-    a            rdf:Property ;
+    a            :cristinfield ;
     rdfs:comment "Free text fore super-users to add explanations/details on why the publication was approved or rejected." ;
```

Comment:
fore -> for

would also extend to "...approved or rejected for NVI reporting."

Because this field is the final verdict, while "merknadtekst" above is the scrap of notes that super-users can write to each other while working on their decision.